docs/changes.rst

```diff
@@ -5,6 +5,8 @@ Pending
 -------
 
 * Removed outdated third-party panels from the list.
+* Do not quote SQL params before passing them to mogrify() for display in SQL
+  panel.
```

Comment:
The spell check failed; maybe something like the following would be a better description of the intent behind the change?

```suggestion
* Avoided the unnecessary work of recursively quoting SQL parameters.
```